proto/astarteplatform/msghub/message_hub_service.proto

```diff
@@ -41,4 +42,10 @@ service MessageHub {
   rpc AddInterfaces(InterfacesJson) returns (google.protobuf.Empty){}
   /* This function should be used to remove one or more interfaces from an instance of the Astarte message hub. */
   rpc RemoveInterfaces(InterfacesName) returns (google.protobuf.Empty){}
+  /* Get properties associated with the passed interfaces. */
+  rpc GetProperties(InterfacesName) returns (StoredProperties){}
+  /* Get all the properties, allowing also filtering by interface ownership. */
+  rpc GetAllProperties(StoredPropertiesFilter) returns (StoredProperties){}
+  /* Get a property defined by the passed interface and path. */
+  rpc GetProperty(PropertyIdentifier) returns (Property) {}
```

Comment:
Is there a way to return a property that is unset? We could mark the Property value field as optional.

Also, we should document how we respond to missing interfaces, like what if we don't have the given interface/path in our introspection. Do we return None or a not found status code?